convert2rhel/data/7/x86_64/configs/oracle-7-x86_64.cfg

```diff
@@ -22,6 +22,7 @@ excluded_pkgs =
   rhnsd
   yum-plugin-ulninfo
   yum-rhn-plugin
+  python-requests
```

Comment:
you may extend the integration test (tests/integration/remove-excluded-pkgs) to cover this case

Reply:
Agree, sounds logical

Reply:
@bocekm  tests included, check if we need to add more checks